index.html

```diff
@@ -92,9 +92,9 @@
     
     <!-- Dismissable notification bar  -->
     <div class="notif-bar-btn" id="close">
-        <div style="display: flex;width:73%;align-items: flex-end;">
+        <div style="display: flex;width:73%;align-items: flex-end; justify-content: center;"><!--To center only the text and the button-->
```

Comment:
Change `align-items: flex-end` to `align-items: center` to nicely centre the elements inside the notif-bar vertically.